R/is_something.R

```diff
@@ -19,30 +19,30 @@ is_parenthesized <- function(expr)
   is.call(expr) && identical(expr[[1L]], quote(`(`))
 }
 
-# Check whether a pipe is a tee.
+# Check whether a pipe is the tee pipe operator.
 #
 # @param pipe A (quoted) pipe
 # @return logical - TRUE if pipe is a tee, FALSE otherwise.
-is_tee <- function(pipe)
+is_tee_pipe <- function(pipe)
 {
   identical(pipe, quote(`%T>%`))
 }
 
-# Check whether a pipe is the dollar pipe.
+# Check whether a pipe is the exposition pipe operator.
 #
 # @param pipe A (quoted) pipe
 # @return logical - TRUE if pipe is the dollar pipe, FALSE otherwise.
-is_dollar <- function(pipe)
+is_exposition_pipe <- function(pipe)
 {
   identical(pipe, quote(`%$%`))
 }
 
-# Check whether a pipe is the compound assignment pipe operator
+# Check whether a pipe is the assignment pipe operator.
```

Comment:
Aren't we removing "operator"?

Reply:
Yes. Sorry. I thought those were descriptive sentences. But it makes more sense to use the new names indeed. Will change.

Reply:
Done.